src/DynamoCore/App.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+  <appSettings>
+    <add key="packageManagerAddress" value="http://107.20.146.184" />
```

Comment:
I'm not entirely happy about having the primary server on HTTPS and the staging on HTTP. OK for now, but lets get this switched over to HTTPS when we can.